### PR TITLE
fix Error handling code

### PR DESCRIPTION
**Pull request by zvezdochiot**
* fix Error handling code
* astyle --style=allman

### CODE REVIEW

File: config.def.h [hunk 1]
```diff
@@ -1,2 +1,2 @@
 /* Comma separated list of file extensions to handle */
-static const char extensions[] = "ff";
+#define extensions "ff"
```
Comment (ids1024) [inline config.def.h:2]: Why use a macro here?

I think it should have the same behavior (the macro could result in multiple copies of the string, but modern compilers probably optimize this). But I think it's generally preferably to not use macros when they don't provide any advantage.

Though maybe I'm missing something.

File: farbfeld.c [hunk 2]
```diff
@@ -62,23 +64,26 @@ static void query(void)
                            load_args,
                            load_return_vals);
 
-    gimp_install_procedure("file_farbfeld_save",
+    gimp_register_file_handler_mime(LOAD_PROC, "image/farbfeld");
```
Comment (ids1024) [inline farbfeld.c:67]: The `"image/farbfeld"` string should probably be in a const or #define as well.

**Review by ids1024 — COMMENTED** [review]
The technical changes here look good.

Though I'm not personally a fan of Allman style. I prefer (among other things) always using attached braces. I suppose consistency with other projects could have some value, but Gimp uses GNU style, and farbfeld is the [suckless style](https://suckless.org/coding_style/).

File: farbfeld.c (hunk 2 above)
Comment (zvezdochiot) [reply farbfeld.c:67]: Optimization for optimization? The bug is fixed, the rest is not important to me.

Not resolved.

File: config.def.h (hunk 1 above)
Comment (zvezdochiot) [reply config.def.h:2]: @ids1024 say> Though maybe I'm missing something.

No, you did not miss anything. This is simple. :)

**Comment by zvezdochiot** [comment]
@ids1024 , use my commit as a template to fix the error on your own.

**Comment by ids1024** [comment]
I've updated gimp-farbfeld to use a clang-format style, and rebased this fix as 8c376162b3037f6be5261cd6ad86d2c9439d01ca.

Thanks!

**Pull request closed without merging**